CHANGELOG.md

```diff
@@ -5,12 +5,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Plugin now can correctly consume provided server certificates
```

Comment:
is it CA certs that it's consuming?

is it more correct to say "plugin supports validation of self-signed certificates using provided `CONJUR_CERT_FILE` env var"? (don't know if I'm understanding how it's configured correctly)

Reply:
also I wish the cert env var was consistent with the rest of our products, but I'll file a follow-up issue for that

Reply:
I'll update though it's not specific to that exact var (the ssl cert can be specified in conjur.conf too)